drivers/blktap2.py

```diff
@@ -1552,7 +1553,9 @@ def activate(self, sr_uuid, vdi_uuid, writable, caching_params):
             # Tapdisk needs the allocation_quantum to be in MB
             options["allocation_quantum"] = int(aq / (1024*1024))
 
-        timeout = util.get_nfs_timeout(self.target.vdi.session, sr_uuid)
+        sr_ref = self.target.vdi.sr.srcmd.params.get('sr_ref')
```

Comment:
If we already have a proper SR object maybe we should use the other_config in there, limiting the number of xapi calls. Not sure we have, though.